lib/util/environment.py

```diff
@@ -51,11 +50,5 @@ def get_test_name():
     name = os.environ.get('TMT_TEST_NAME')
     if name:
         return name
-    # under Restraint (Beaker, OSCI, etc.)
-    name = os.environ.get('RSTRNT_TASKNAME', '')
```

Comment:
This is not just Beaker native API, but also for OSCI, which would break with this removal.